tests/run-tests

```diff
@@ -204,39 +204,39 @@ def run_tests(pyb, tests, args):
     skip_const = False
 
     # Check if micropython.native is supported, and skip such tests if it's not
-    native = run_micropython(pyb, args, 'feature_check/native_check.py')
+    native = run_micropython(pyb, args, base_path + '/feature_check/native_check.py', True)
     if native == b'CRASH':
         skip_native = True
 
     # Check if arbitrary-precision integers are supported, and skip such tests if it's not
-    native = run_micropython(pyb, args, 'feature_check/int_big.py')
+    native = run_micropython(pyb, args, base_path + '/feature_check/int_big.py', True)
     if native != b'1000000000000000000000000000000000000000000000\n':
         skip_int_big = True
 
     # Check if set type (and set literals) is supported, and skip such tests if it's not
-    native = run_micropython(pyb, args, 'feature_check/set_check.py')
+    native = run_micropython(pyb, args, base_path + '/feature_check/set_check.py', True)
     if native == b'CRASH':
         skip_set_type = True
 
     # Check if async/await keywords are supported, and skip such tests if it's not
-    native = run_micropython(pyb, args, 'feature_check/async_check.py')
+    native = run_micropython(pyb, args, base_path + '/feature_check/async_check.py', True)
     if native == b'CRASH':
         skip_async = True
 
     # Check if const keyword (MicroPython extension) is supported, and skip such tests if it's not
-    native = run_micropython(pyb, args, 'feature_check/const.py')
+    native = run_micropython(pyb, args, base_path + '/feature_check/const.py', True)
     if native == b'CRASH':
         skip_const = True
 
     # Check if emacs repl is supported, and skip such tests if it's not
-    t = run_micropython(pyb, args, 'feature_check/repl_emacs_check.py')
+    t = run_micropython(pyb, args, base_path + '/feature_check/repl_emacs_check.py', True)
     if not 'True' in str(t, 'ascii'):
         skip_tests.add('cmdline/repl_emacs_keys.py')
 
-    upy_byteorder = run_micropython(pyb, args, 'feature_check/byteorder.py')
-    has_complex = run_micropython(pyb, args, 'feature_check/complex.py') == b'complex\n'
-    has_coverage = run_micropython(pyb, args, 'feature_check/coverage.py') == b'coverage\n'
-    cpy_byteorder = subprocess.check_output([CPYTHON3, 'feature_check/byteorder.py'])
+    upy_byteorder = run_micropython(pyb, args, base_path + '/feature_check/byteorder.py')
+    has_complex = run_micropython(pyb, args, base_path + '/feature_check/complex.py') == b'complex\n'
```

Comment:
Why doesn't this have True for the is_special arg?  (some with upy_byteorder test)